ui/views/base.jsx

```diff
@@ -60,7 +60,7 @@ const Layout = ({
       footerLinks={footerLinks}
       phaseBanner={{
         phase: 'beta',
-        feedbackUrl: '/feedback'
+        content: 'Your opinions make things better. Share them in [this short survey](https://www.homeofficesurveys.homeoffice.gov.uk/s/D6XIF5/).'
```

Comment:
```suggestion
        content: <span>Your opinions make things better. Share them in <a href="https://www.homeofficesurveys.homeoffice.gov.uk/s/D6XIF5/" target="_blank">this short survey</a>.</span>
```

Would this not remove the need for the whole markdown thing?

Reply:
nah cos react innit

Reply:
Or to be honest, just code the thing into the phase banner component. It's not as if any of this stuff is actually re-used anywhere.

Reply:
you'd have to do something mad like `__dangerouslySetInnerHTML`

Reply:
OIC - not a string. pls hld.